react/Gallery.js

```diff
@@ -13,13 +14,32 @@ import searchResult from './searchResult.css'
 /**
  * Canonical gallery that displays a list of given products.
  */
+
+const TWO_ITEMS = 2
```

Comment:
Could you improve those variable names? Maybe put `TWO_ROW_ITEMS` or something alike for better readability.